### DISCUSSION TITLE
Cant find the launcher for Inventory Kamera..

### DISCUSSION THREAD

**Discussion by xxeverlee** [Andrewthe13th/Inventory_Kamera#365] [Q&A] [answered]
Exactly what the title says.. Whats the name of the launcher? I have all the files, its just that I don't know which one is the launcher that runs the program-

**Comment by xxeverlee** [comment, marked as answer]
Ah, nevermind. I am an idiot and didnt download the latest release :I

**Reply by TriPesto** [reply]
Hey, I have the newest version, yet I cant find how to open the program, how did you do it? I looked inside the .zip file and I didn't find an .exe or any kind of launcher. Any help would be great.  (Inventory_Kamera-1.3.7)
